src/tokens.ts

```diff
@@ -137,6 +137,18 @@ export const TOKEN_SYMBOLS_MAP = {
     },
     coingeckoId: "gho",
   },
+  GRASS: {
+    name: "Grass",
+    symbol: "GRASS",
+    decimals: 18,
+    addresses: {
+      // [CHAIN_IDs.LENS]: @todo
+      // [CHAIN_IDs.LENS_SEPOLIA]: @todo
+      // [CHAIN_IDs.MAINNET]: @todo
+      [CHAIN_IDs.SEPOLIA]: "0x8D725d9dBBb5E0667efeDC833D6A9e8C6cA02C68",
```

Comment:
tbd on this - the contract on Sepolia has 0 transactions ever and looks kinda ded.

https://sepolia.etherscan.io/address/0x8d725d9dbbb5e0667efedc833d6a9e8c6ca02c68